app/routes/profiles.rb

```diff
@@ -21,14 +21,21 @@ def protected!
 
       get "/settings/:user/trial" do
         @user = params[:user]
-        orgs = gh.user.memberships.orgs("active").body
-        not_admin = orgs.select!{|org| org["role"] == "admin"}.empty?
+
+        account_type = gh.users(@user)["type"]
+        if account_type == "User"
+          is_admin = gh.user["login"] == @user
+        elsif account_type == "Organization"
+          orgs = gh.user.memberships.orgs("active").body
```

Comment:
the body call doesn't make sense, unless this endpoint is slightly different in ghee